tools/serverpod_cli/lib/src/analyzer/models/yaml_definitions/class_yaml_definition.dart

```diff
@@ -67,6 +67,11 @@ class ClassYamlDefinition {
                 valueRestriction:
                     restrictions.validateRelationInterdependencies,
                 allowEmptyNestedValue: true,
+                mutuallyExclusiveKeys: {
+                  Keyword.defaultKey,
+                  Keyword.defaultModelKey,
+                  Keyword.defaultDatabaseKey,
+                },
```

Comment:
We cant add test for this for now, because defaults are only supported for DateTime fields, and relation is only supported for classed and int types, and its impossible to add tests now, maybe in the near future yes.